Gemfile

```diff
@@ -23,4 +24,5 @@ group :development do
   gem 'capistrano-rails', require: false
   gem 'capistrano-passenger', require: false
   gem 'pry-byebug'
+  gem 'timecop', '~> 0.9.1'
```

Comment:
Timecop is used in seeding to simulate incidents having been created/updated at different times in the past.